pygmt/src/meca.py

```diff
@@ -211,6 +215,21 @@ def meca(
         to the circle. Use **+s**\ *size* to set the diameter of the circle
         [Default is no circle]. Use **+p**\ *pen* to set the line pen
         attributes [Default is 0.25p].
+    compressionfill : str
+        Set color or pattern for filling compressive quadrants
+        [Default is black].
+    extensionfill : str
+        Set color or pattern for filling extensive quadrants
+        [Default is white].
+    pen : str
+        Set pen attributes for outline of beachball
+        [Default is ``"0.25p,black,solid"``].
+    cmap : str
+        File name of a CPT file or a series of comma-separated colors (e.g.,
+        *color1,color2,color3*) to build a linear continuous CPT from those
+        colors automatically. The color of the compressive quadrants is
+        determined by the z-value (i.e., event depth or the third column for
+        an input file).
     no_clip : bool
         Does NOT skip symbols that fall outside frame boundary specified by
```

Comment:
```suggestion
        Do NOT skip symbols that fall outside frame boundary specified by
```

Reply:
I agree with this change by @michaelgrund!
Maybe we should decide, whether we want to use either the general verb form or the third-person form in the docstrings? This is currently not consistent across the documentation, even not within a single function / method, e.g., here for `compressionfill`, `extensionfill`, and `pen` it is `set`, but for `offset` and `no_clip` it is (was) `offsets` and `does`.

Reply:
I think GMT decided to use the general verb form some years ago (see changes in https://github.com/GenericMappingTools/gmt/commit/a57fd6c2f5b16c7fcaaaaa4b806233d19f0cd75b and https://github.com/GenericMappingTools/gmt/commit/8256073836925692451f5ece92e2cb81ec7fad26)

Reply:
Thanks for pointing out these upstream PRs. I started working on this in PR #2376.